src/main/java/pages/LoginPage.java

```diff
@@ -10,22 +11,52 @@
     private By passwordLocator = By.id("passwd");
     private By forgotPasswordLink = By.className("lost_password");
     private By signInButton = By.id("SubmitLogin");
+    private By logInErrorMessage = By.cssSelector("div.alert-danger li");
 
-    public LoginPage(WebDriver driver) {
-        super(driver);
+    public LoginPage(WebDriver driver, Logger log) {
+        super(driver, log);
     }
 
+    /** login user using Sign In button */
     public MyAccountPage logInUserSignInBtn(String email, String password) {
+        log.info("Executing LogIn test using SignIn button with email " + email + " and password " + password);
         type(email, emailLocator);
         type(password, passwordLocator);
         click(signInButton);
-        return new MyAccountPage(driver);
+        return new MyAccountPage(driver, log);
     }
 
+    /** login user using Enter key */
     public MyAccountPage logInUserEnterKey(String email, String password) {
+        log.info("Executing LogIn test using Enter Key with email " + email + " and password " + password);
         type(email, emailLocator);
         type(password, passwordLocator);
         pressKey(passwordLocator, Keys.ENTER);
-        return new MyAccountPage(driver);
+        return new MyAccountPage(driver, log);
+    }
+
+    /**login user using invalid email/ password */
+    public void negativeLogIn(String email, String password) {
```

Comment:
what is the scope of this method? it seems to do exactly the same actions as 'logInUserSignInBtn' method

Reply:
Will be resolved in next PR - Refactored NegativeLoginTest to also use logInUserSignInBtn method